src/balancerd/src/service.rs

```diff
@@ -86,6 +83,30 @@ pub struct Args {
         requires = "frontegg-resolver-template"
     )]
     frontegg_admin_role: Option<String>,
+
+    /// An SDK key for LaunchDarkly.
+    ///
+    /// Setting this will enable synchronization of LaunchDarkly features.
+    #[clap(long, env = "LAUNCHDARKLY_SDK_KEY")]
+    launchdarkly_sdk_key: Option<String>,
+    /// The duration at which the LaunchDarkly synchronization times out during startup.
```

Comment:
There's a big comment in envd explaining that we require an LD sync the first time an env starts (because the defaults in code could be arbitrarily out of date) but that we allow timing out after the first boot (so that LD isn't a tier 1 dep and whatever was stored in the catalog from the previous boot is probably fine). https://github.com/MaterializeInc/materialize/blob/main/src/adapter/src/coord.rs#L3816-L3854

Of course, there's no catalog in balancerd, so we can't do the same. The policy you have in the PR seems fine to me, but could be getting a quick @benesch feels on the tradeoffs.

Reply:
Seems fine to me too!

Reply:
Added a buncha comments scaring us into choosing good default values so if LD is down we still start and things are reasonable.